frontend/src/metabase/metadata/pages/DataModel/components/FieldSection/FieldSection.tsx

```diff
@@ -63,3 +81,6 @@ export const FieldSection = ({ databaseId, field }: Props) => {
     </Stack>
   );
 };
+{
+  /* <Box className={S.container} h="100%" pb="lg" px="xl"> */
```

Comment:
Commented code is left in for debugging?

Reply:
a0cb49fc1f409d1c70c613088f0110e7674824f5